app/presenters/publishing_api_presenters/edition.rb

```diff
@@ -37,10 +37,6 @@ def as_json
 
   private
 
-    # An incomplete and temporary details hash that only currently includes
```

Comment:
What was the reason for removing this part of the comment? I think it was intended to explain the reason why these placeholder items contain a details hash when usually they don't.